src/components/tag/Tag.tsx

```diff
@@ -0,0 +1,29 @@
+import clsx from "clsx";
+import React from "react";
+
+type TagProps = {
+  children: React.ReactNode;
+  onClick?: () => void;
+  selected?: boolean;
+};
+
+const Tag: React.FunctionComponent<TagProps> = ({
+  onClick,
+  selected,
+  children
+}) => {
+  return (
+    <button
+      type="button"
+      className={clsx(
+        "tag tag--outlined",
+        selected && "tag--outlined-selected"
+      )}
+      onClick={onClick ?? undefined}
```

Comment:
Is it necessary to specify undefined here? The prop type already indicates that it can be undefined.